openshift/deployment.yaml

```diff
@@ -0,0 +1,149 @@
+apiVersion: v1
+kind: Template
+metadata: {}
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    creationTimestamp: null
+    name: tomcat
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: tomcat
```

Comment:
Introduce `NAME` and `SUFFIX` parameters and use them in .metadata.name and related fields.